thirdeye/thirdeye-frontend/yarn.lock

```diff
@@ -4974,6 +4974,14 @@ js-yaml@^3.12.0, js-yaml@^3.2.5, js-yaml@^3.2.7, js-yaml@^3.3.0, js-yaml@^3.9.1:
     argparse "^1.0.7"
     esprima "^4.0.0"
 
+js-yaml@^3.13.1:
```

Comment:
What is this file used for?

Reply:
This parses yaml to JSON.  It was able to parse the valid yaml that yamljs was failing on.  So I installed it as a backup parser.

Reply:
I mean is this file generated  automatically? Should we put it into git?

Reply:
Yes, this file is generated so that yarn knows which versions and packages to install for Thirdeye's frontend, when the user runs 'yarn install.'